codecov_auth/views/okta.py

```diff
@@ -96,12 +103,18 @@ def _store_iss_cookie(self, iss: str, response: HttpResponse):
 
     def _perform_login(self, request: HttpRequest) -> HttpResponse:
```

Comment:
i think after we've successfully logged in we have to call `self.remove_state(state)` ([see the github view for reference](https://github.com/codecov/codecov-api/blob/f85b724284038271b3b68d8d968b633fd29eca5c/codecov_auth/views/github.py#L127))